content/chronograf/v1.9/about_the_project/release-notes-changelog.md

```diff
@@ -8,7 +8,31 @@ menu:
     parent: About the project
 ---
 
+## v1.9.3 [2022-02-01]
+
+### Features
+- Add ability to rename TICKscripts.
+- Changes to the `Queries` page:
+  - Add `CSV download` button. 
+  - Rename `Running` column to `Duration`.
+  - Add `Status` column. When hovering over the `Duration` column, status shows `Kill` confirmation button.
+  - Modify the `CSV` export to include the `Status` column.
+- Upgrade to use new `google.golang protobuf` library.
+
+### Bug Fixes
+- Log remote URL when ping fails.
+- Repair retrieval of background job results. Allows for the database to wait to store response before alerting the 
```

Comment:
This can be dropped.  This is a defect fix which is the reason why 1.9.2 didn't get released officially.

Reply:
first 2 bullet points removed